src/caret_analyze/plot/stacked_bar/latency_stacked_bar.py

```diff
@@ -89,7 +89,7 @@ def to_stacked_bar_data(
         """
         response_records: RecordsInterface = \
             self._get_response_time_record(self._target_objects)
-        stacked_bar = StackedBar(response_records, converter)
+        stacked_bar = StackedBar(response_records)
```

Comment:
By this change, `converter` argument of `to_stacked_bar_data` is not used anymore. That means `StackedBarPlot.to_dataframe` does not support `sim_time`. Can you check if `StackedBarPlot.to_dataframe` generates same results for `system_time` and `sim_time`? If that is the case, I suggest:
- reverting this change and changes of `record/records_service/stacked_bar.py` (After reverting, I think `start time` of results of `to_stacked_bar_data` are converted.)
- removing conversion of `start time` from `StackedBarSource`.

Reply:
Your point is correct.
In the process of fixing it, we discovered a bug in the sim_time conversion of to_dataframe() in staced_bar in PR#433, which caused the time axis to not be converted to sim_time.